sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOReadTest.java

```diff
@@ -17,8 +17,6 @@
  */
 package org.apache.beam.sdk.io;
```

Comment:
High level note - we'll need to be careful when merging this with the dynamic TextIO write, as a merge like this (where code has migrated files) is very error prone. Though hopefully my changes will end up being only in the Write file.